static/index.html

```diff
@@ -41,8 +41,8 @@ <h1 class="title">Remote Play</h1>
       </button>
       <div class="touchpad" id="touchpad">
         <p>
-          Use this area as a touchpad.<br />Supported gestures:<br />track and
-          tap (Left Mouse Button).
+          Use this area as a touchpad.<br />Supported gestures:<br />Track and
+          Tap (Left Mouse Button).<br/>Double Tap (Right Mouse Button).
```

Comment:
Let's break this into multiple lines for readability. Also I don't think we should have a line break before `Double tap`.
```html
Use this area as a touchpad.<br/>
Supported gestures:<br/>
Track, tap (left mouse button), and double tap (right mouse button).
```

Reply:
Ok I'll do that..